tests/integration/route_test.py

```diff
@@ -2437,3 +2437,71 @@ def test_add_route_with_advmss(eth1_up):
     )
     cur_state = libnmstate.show()
     assert_routes(routes, cur_state)
+
+
+# https://issues.redhat.com/browse/RHEL-107130
+@pytest.mark.tier1
+def test_apply_ignored_routes_to_ignored_iface(eth1_static_ip):
```

Comment:
What happens if you try to apply a route without state to an ignored interface? We should add a test to see that an error is raised.

And what happens if you apply a route with `state: absent`? A test is also desirable (if it doesn't exist yet)